### PR TITLE
remove charge prices

### PR DESCRIPTION
**Pull request by Issa-morad**
## Description
Briefly descripe the PR.

## Screenshots
Add if necessary.
Copy paste should work.

## Type of PR
- [ ] Feature
- [x] Bugfix
- [ ] Documentation
- [ ] Test
- [ ] Code refactor

## Did you write tests?
- [ ] Yes
- [x] No
- [ ] No, I need help

## If you wrote tests, did they pass?
- [ ] Yes
- [x] No
- [ ] No, I need help

## In which environment did you test?
- [x] Development
- [ ] Production (backend API)
- [ ] Didn't test

### example steps (change for your PR):
- [ ] Click on the Active Chargers button
- [ ] Try to edit a charger
- [x] Refresh the page
- [x] Make sure that it works

### CODE REVIEW

**Comment by Issa-morad** [comment]
remove charge point price field

File: react-app/src/dependencyContainer.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import { createContainer, asClass, Lifetime, InjectionMode } from 'awilix';
 // change /mock/ to /remote/ to use api data
 
-import AuthenticationProvider from './remote-access/remote/AuthenticationProvider';
+import AuthenticationProvider from './remote-access/mock/AuthenticationProvider';
```
Comment (WinterBoii) [inline react-app/src/dependencyContainer.ts:6]: Dont commit dependancy container if changing from mock to remote and vice versa

**Review by WinterBoii — DISMISSED** [review]

File: react-app/src/dependencyContainer.ts (hunk 1 above)
Comment (Issa-morad) [reply react-app/src/dependencyContainer.ts:6]: ok

**Review by vikjoh1 — CHANGES_REQUESTED** [review]
Looks good! But you should also remove the price field in the charger-stations mock data.

**Review by vikjoh1 — APPROVED** [review]
LGTM!

**Pull request merged**